README.md

```diff
@@ -1,25 +1,115 @@
+are a hoped-for feature.
+
+* Providing DPI information to the application.
+
+* Clipboard.
+
+* File dialog.
+
+The general philosophy is that a task is in scope if it requires deep integration
+with the platform and is not easy to separate out as a separate library that layers
+on top of Glazier.
+
+## Hooks provided
+
+Glazier does not provide drawing primitives and is intended to be agnostic to
+the drawing infrastructure. It uses [raw-window-handle] to provide an attachment
+point for the drawing code, a widely used abstraction in the Rust ecosystem. A
+top priority will be integrating with the [wgpu] ecosystem. In addition, we would
+gladly accept integration work to make [Piet] run on top of Glazier, but this is
+not a core priority.
+
+We hope to integrate with [AccessKit] for accessibility.
+
+While Glazier currently has primitive support for scheduling repaint cycles,
+ultimately we would like to support [frame pacing]. Doing the actual decision
+of when to repaint is probably out of scope, but providing portable infrastructure
+([CVDisplayLink] on macOS, presentation statistics, scheduling based on high resolution
+timers) is in scope.
+
+## Out of scope
+
+We have no solution to interfacing with the system compositor. This is necessary
+to handle embedded video content properly, and is also a good way to stitch
+together other embedded content such as web views.
+
+Like drawing, most font and text issues are out of scope. Localization and
+internationalization are expected to be handled by the layer above, though platform
+hooks would be in scope (for example, querying the locale preference).
+
+Glazier does not provide a solution for packaging and distribution of applications.
+Work on this is needed, and we would gladly cooperate with such efforts.
 
 ## Design
 
-The code in `glazier` can be divided into roughly two categories: the
+The code in Glazier can be divided into roughly two categories: the
 platform agnostic code and types, which are exposed directly, and the
 platform-specific implementations of these types, which live in per-backend
 directories in `src/backend`. The backend-specific code for the current
 backend is reexported as `glazier::backend`.
 
-`glazier` does not generally expose backend types directly. Instead, we
+Glazier does not generally expose backend types directly. Instead, we
 expose wrapper structs that define the common interface, and then call
 corresponding methods on the concrete type for the current backend.
 
 ## Unsafe
 
 Interacting with system APIs is inherently unsafe. One of the goals of
-`glazier` is to handle all interaction with these APIs, exposing
-a safe interface to `druid` and other possible consumers.
+Glazier is to handle all interaction with these APIs, exposing
+a safe interface to [Druid] and other possible consumers.
+
+## Similar libraries
+
+* [winit]. This is by far the most commonly used window creation library. As
```

Comment:
> most commonly used window creation library

My suspicion would be that if you go outside of Rust, you'll find a more commonly used window creation library. But this isn't an important point.
Perhaps `similar crates` would be a more precise heading.